### DISCUSSION TITLE
加载tidyverse包错误

### DISCUSSION THREAD

**Discussion by Renmr488** [ShixiangWang/self-study#105] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/62462250/195233959-9efb9db1-8cc6-4b4f-b217-71680b06dc99.png)
请教大佬，加载tidyverse包时，遇到了这个问题，如何解决？

**Comment by ShixiangWang** [comment, marked as answer]
重启R，然后安装rlang最新版本。

**Reply by Renmr488** [reply]
谢谢，已解决
